roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+- name: Verify Ansible Service Broker project name exists
+  block:
+    - name: Get Ansible Service Broker project name
+      shell: oc get projects | grep -e ansible-service-broker -e automation-service-broker | awk '{print $1}'
+      register: broker_project_name
+      failed_when: broker_project_name.stdout == ""
+    - set_fact:
+        broker_project_name: '{{ broker_project_name.stdout }}'
+  rescue:
+    - name: Fail when Ansible Service Broker project was not found
+      pause:
+        prompt: |
+          Ansible Service Broker project not found.
+          Please verify that currently logged in user has privileges to list and edit Ansible Service Broker Project.
+
+          Hit Ctrl + c and then 'a' to abort the installation
+
+- name: Get Ansible Service Broker Configmap
+  shell: oc get configmap broker-config -o jsonpath='{.data.broker-config}' -n {{ broker_project_name }}
+  register: original_configmap_raw
+
+- set_fact:
+    original_configmap: '{{ original_configmap_raw.stdout | from_yaml }}'
+
+- name: Apply configuration of Broker necessary for Mobile Services
+  set_fact:
+    original_configmap: "{{ original_configmap|combine({ item.section: { item.name: item.value }}, recursive=True) }}"
+  with_list: "{{ broker_mobile_configuration }}"
+
+- name: Get additional config of broker registry (from ../files)
+  set_fact:
+    additional_registry_config: "{{ lookup('file', '../files/aerogearcatalog_registry.yaml') | from_yaml }}"
+
+- set_fact:
+    additional_registry_config: "{{ additional_registry_config.registry }}"
+
+- set_fact:
+    new_registry_config: "{{ original_configmap.registry }}"
+
+- name: Add broker registry configuration from file (if it's not already present in original configmap)
+  set_fact:
+    new_registry_config: '{{ new_registry_config + [item] }}'
+  with_list: "{{ additional_registry_config }}"
+  when: item not in new_registry_config
+
+- set_fact:
+    new_configmap: "{{ original_configmap|combine({ 'registry': new_registry_config }, recursive=True) }}"
+
+- name: Patch existing configmap
+  block:
+    - set_fact:
+        new_configmap_raw: "{{ new_configmap | to_nice_yaml | replace('\n', '\\n') }}"
+    - name: Patch Broker Configmap
+      command: oc patch configmap broker-config -p "{\"data\":{\"broker-config\":\"{{ new_configmap_raw }}\" }}" -n {{ broker_project_name }}
```

Comment:
it looks like this doesn't trigger a re-deploy of the ASB. We might need to redeploy the ASB again.

Reply:
Great catch, will fix it in a sec